### DISCUSSION TITLE
Initial window not close after open one file through nerdtree

### DISCUSSION THREAD

**Discussion by rafael-hs** [goolord/alpha-nvim#195] [Q&A]
Hey guys, any one know how I can change this behaviour? I have the same issue using dashboard nvim plugin

https://user-images.githubusercontent.com/26948732/226793477-bae6c2cb-1a33-4549-803b-a96fff41216d.mov

**Comment by rafael-hs** [comment]
anyone?

**Comment by ayamir** [comment]
It also appears for me, but for the file opened in quickfix list.

**Reply by malishengo** [reply]
how do you fix it?

**Reply by rafael-hs** [reply]
@ayamir @malishengo Did you manage to fix it?
